app/client/templates/layouts/master_layout/master_layout.js

```diff
@@ -45,5 +47,36 @@ Template.MasterLayout.events({
 
   'click [data-login]': function (e, tmpl) {
     Meteor.loginWithMeteorDeveloperAccount();
+  },
+
```

Comment:
The following should probably need to be moved to the search template (client/search.js).